packages/rocketchat-oembed/client/oembedImageWidget.html

```diff
@@ -7,7 +7,7 @@
 			{{#if loadImage}}
 				<figure>
 					<div class="inline-image" style="background-image: url('{{url}}');">
-						<img src="{{url}}" rel="noopener noreferrer" height="200" class="gallery-item">
```

Comment:
This causes the lazyloading images to resize when they load in. In other words, the images first display at the tiny lazyload size, then spring up in size to their full size:

![2018-06-22_12-50-33](https://user-images.githubusercontent.com/39674991/41773126-4034052e-761b-11e8-8286-c8d492a57cf7.gif)

With lots of images, this sort of jumping around can get quite messy.
![2018-06-22_12-51-37](https://user-images.githubusercontent.com/39674991/41773129-42c7e3aa-761b-11e8-813b-6e292e26b89c.gif)

Reply:
Ah, I see. Do you have more information on how the lazy loading mechanism works? I looked through the source code but haven't been able to come to a conclusion and even when throttling my connection it loads too quickly to be able to inspect the unloaded version in my browser. I just see some JavaScript that grabs the data-src attribute of the low-quality image to load the full quality one, but not how the low-quality one is generated and what metadata may be available.

https://github.com/RocketChat/Rocket.Chat/blob/c5c65bd53a0d0bf5bb77ce7be3f703e2d766f310/packages/rocketchat-lazy-load/client/lazyloadImage.html#L2 implies that a height is known, but the gif you attached implies it is not actually set (or the CSS should have dealt with this situation just fine).

Reply:
Well, with a bit of research I've found two things:
- `height` seems to always be set to 200 no matter what
- `height` is overridden anyway by the CSS `height: auto;`

It looks like theres a clear issue with the way `height` is calculated.
It's inserted [here](https://github.com/RocketChat/Rocket.Chat/blob/c5c65bd53a0d0bf5bb77ce7be3f703e2d766f310/packages/rocketchat-message-attachments/client/messageAttachment.html#L69) and calculated [here](https://github.com/RocketChat/Rocket.Chat/blob/20db19bbf3e865fcc04de7493b011d4cb56a5416/packages/rocketchat-file-upload/server/methods/sendFileMessage.js#L40), but I don't see why it's not sending it correctly...

If you want to throttle the loading, you can use Chrome's Network tab (F12 -> Network) and just click "Offline" as soon as an image tries to load (for example, you can refresh, and turn on Offline when the chat loads but the images have yet to). Do turn on "Disable Cache" though, else it'll only work once.

Reply:
@vynmera I think it's safe to omit the `height` attribute and set the `200px` on CSS. What do you think?

Reply:
@tassoevan That would make images under 200px tall scale up (which is what this PR aims to resolve), at least for the lazyload, no?

Reply:
@vynmera You're right :see_no_evil: I'll take a look at the preview creation. I put a breakpoint where you pointed out and the debugger did nothing...

Reply:
@rodrigok After reviewing [this](https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/rocketchat-file-upload/server/lib/FileUpload.js#L165), I realized that the upload image dimensions are lost, therefore it's impossible to merge this PR. Can you help me here?

Reply:
@tassoevan we need to rethink that part, suggestions?

Reply:
@rodrigok One more thing that I discovered: despite adding the dimensions to the `rocketchat_uplodas` collection, the message attachments in `rocketchat_messages` have only a subset of the fields.

Reply:
So, this is it: without the dimensions, which are not appended to the message attachment, it's impossible to determine the right size of the preview image.

Reply:
Hasn't that been resolved with #11521?

Reply:
@TheLastProject (I apologize for my absence here.) It was partially solved; a stealthy small bug in Apps Engine is wiping out the attachment data in some circumstances. @d-gubert and @rodrigok told me what is happening and we're discussing what to do.